### PR TITLE
Page Layouts: Improve RTL compatibility

### LINKED ISSUE #5909: Page Layouts : switching to RTL alters the layout preview
## Bug Description

Switching to RTL is changing the presentation of some of the layouts.

eg: default (RTL) layout preview
Screen Shot 2021-01-12 at 4 06 40 PM

switched to RTL layout preview - note the Sun Protection done right layout has changed!
Screen Shot 2021-01-12 at 4 07 03 PM

Seeing the layout applied to page in editor:
Screen Shot 2021-01-12 at 4 08 47 PM

## Expected Behaviour

The preview of the page layout should consistently reflect what will be applied to the page.

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

- See thread below and Pascal's shared solution for how the editor handles this issue. 
- We should wrap both templates in dashboard as well as page layouts to prevent this.
- Add tests

### PR DESCRIPTION
**Pull request by zachhale**
## Context

Page Layouts and Dashboard template previews are inconsistent with the editor in RTL mode.

## Summary

Brings Page Layouts and Dashboard Template previews in line with what they look like when opened in the editor.

## Relevant Technical Choices

Testing this highlighted that some text elements at least from templates render differently in the editor in RTL mode than when they are published/previewed. This issue is represented in a new ticket #6068.

There are no Karma tests set up for Page Layouts yet to snapshot this, so I'm deferring tests to the in progress PR: #5945. I'm not sure what would make sense to test here other than snapshot.

## Testing Instructions

Turn on RTL mode and ensure that the page layouts in the pane are consistent with what appears when you click to apply them to the page.

---

Fixes #5909

### CODE REVIEW

**Comment by swissspidy** [comment]
Curious: does this also affect text sets in the editor?

**Comment by zachhale** [comment]
Ok so I just went and did some more testing and now I'm not seeing what I thought was inconsistency and am seeing the same weird text output in the published story. Maybe something was weird with me opening RTL in one browser and not the other? I'm confused. Would appreciate another set of eyes on this. I see text sets look differently in RTL, but they apply and publish how they show up in the text sets pane so 🤷 .

File: assets/src/edit-story/components/previewPage/previewPage.js [hunk 1]
```diff
@@ -115,20 +115,24 @@ function PreviewPage({
   onAnimationComplete,
   subscribeGlobalTime,
 }) {
+  // Preview is wrapped in StyleSheetManager w/ stylisPlugins={[]} in order to prevent
+  // elements from shifting when in RTL mode since these aren't relevant for story previews
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/previewPage/previewPage.js:119]: This is a great note, thank you for adding it!

**Review by BrittanyIRL — APPROVED** [review]
LGTM! Template pages in the page layouts section of the editor and the templates view of the dashboard are consistent in direction regardless of RTL or LTR setting.

**Review by swissspidy — APPROVED** [review]

**Pull request merged**